src/fairchem/core/calculate/ase_calculator.py

```diff
@@ -93,11 +93,25 @@ def __init__(
 
         self.predictor = predict_unit
 
+        if predict_unit.inference_settings.external_graph_gen is True:
+            r_edges = True
+            max_neigh = 300
+            radius = 6.0  # Default radius for edge generation
```

Comment:
Does it make sense to pull this from the backbone? its crossing a boundary , but at least its in sync , vs falling out of sync?